lib/src/nakama_client/nakama_grpc_client.dart

```diff
@@ -368,4 +368,22 @@ class NakamaGrpcClient extends NakamaBaseClient {
       options: _getSessionCallOptions(session),
     );
   }
+
+  @override
+  Future<void> listLeaderboardRecords({
+    required model.Session session,
+    required String leaderboardId,
+    List<String>? ownerIds,
+    int? limit,
+    String? cursor,
+    String? expiry,
+  }) {
+    return _client.listLeaderboardRecords(ListLeaderboardRecordsRequest(
+      leaderboardId: leaderboardId,
+      ownerIds: ownerIds,
+      limit: limit, // Int32Value?
+      cursor: cursor,
+      expiry: expiry, // Int64Value?
+    ));
```

Comment:
I couldn't really find an example how the conversion should work, but when running the tests I get issues like:
`Error: The argument type 'int?' can't be assigned to the parameter type 'Int32Value?'.`